confgenerator/confgenerator.go

```diff
@@ -32,6 +32,8 @@ import (
 	"github.com/shirou/gopsutil/host"
 )
 
+const FluentBitSelfLogTag = "ops-agent-fluent-bit"
```

Comment:
This should be unexported

Reply:
Changed in: [8145132](https://github.com/GoogleCloudPlatform/ops-agent/pull/1040/commits/81451323dc4ba31fa2296dcc8c76b75838800967)